package.json

```diff
@@ -2,10 +2,11 @@
   "name": "@farbenmeer/react-spring-slider",
   "version": "1.3.5",
   "main": "dist/index.js",
+  "license": "MIT",
   "author": {
-    "name": "Max Strübing",
-    "email": "max.struebing@farbenmeer.de",
-    "url": "https://maex.me"
+    "name": "farbenmeer GmbH",
+    "email": "kontakt@farbenmeer.de",
+    "url": "https://farbenmeer.de/"
```

Comment:
As he does no longer maintain it and it lives in the farbenmeer-namespace anyway I guess...